ql/src/java/org/apache/hadoop/hive/ql/ddl/table/info/desc/DescTableOperation.java

```diff
@@ -148,7 +148,8 @@ private void getColumnsNoColumnPath(Table table, Partition partition, List<Field
 
     // Fetch partition statistics only for describe extended or formatted.
     if (desc.isExtended() || desc.isFormatted()) {
-      if (table.isPartitioned() && partition == null) {
+      boolean shouldGetPartStats = MetastoreConf.getBoolVar(context.getConf(), MetastoreConf.ConfVars.DESCTABLE_ENABLE_PARTITION_STATS);
+      if (table.isPartitioned() && partition == null && shouldGetPartStats) {
```

Comment:
Need to add a unit test to verify the output with and without this config is set.

Reply:
Added the tests. Let me know if I did it right. Here are the steps I did

1. `mvn test -Dtest=TestCliDriver -Dqfile=describe_table.q -Dtest.output.overwrite=true`
2. It actually made a new output file and was in `ql/src/test/results/clientpositive/`
3. I saw `describe_table.q.out` in the OSS is actually in a different path `ql/src/test/results/clientpositive/llap`. What I did was to move the new output file to the correct location and that's how I got the final diff.

Reply:
If you used TestMiniLlapCliDriver instead of TestCliDriver, then output would've been created under ql/src/test/results/clientpositive/llap